.github/workflows/workflow.yml

```diff
@@ -207,6 +207,38 @@ jobs:
         if: matrix.os == 'macOS-latest'
         run: brew install mercurial
 
+      # Install Pijul deps
+      - name: Setup | Pijul deps (linux)
+        if: matrix.os == 'ubuntu-latest'
+        run: sudo apt install make libsodium-dev libclang-dev pkg-config libssl-dev libxxhash-dev libzstd-dev clang
+
+      - name: Setup | Pijul deps (macos)
+        if: matrix.os == 'macOS-latest'
+        run: brew install llvm libsodium openssl xxhash zstd
+
+      - name: Install vcpkg deps
+        if: matrix.os == 'windows-latest'
+        shell: bash
+        run: |
+          $VCPKG_INSTALLATION_ROOT/vcpkg.exe install --triplet=x64-windows-static-md xxhash zstd openssl openssl-windows
+          cd $VCPKG_INSTALLATION_ROOT/installed/x64-windows-static-md/lib
+          cp zstd_static.lib zstd.lib
+
+      # Install Pijul
+      - name: Setup | Pijul (windows)
+        if: matrix.os == 'windows-latest'
+        shell: bash
+        run: |
+          export VCPKG_ROOT=$VCPKG_INSTALLATION_ROOT
+          export LIB=$VCPKG_ROOT/installed/x64-windows-static-md/lib
+          export OPENSSL_DIR=$VCPKG_ROOT/installed/x64-windows-static-md
+          cargo install pijul --version "~1.0.0-beta" --locked
+
+
+      - name: Setup | Pijul
+        if: matrix.os == 'macOS-latest' || matrix.os == 'ubuntu-latest'
+        run: cargo install pijul --version "~1.0.0-beta" --locked
```

Comment:
In think this much set-up code and compilation time isn't worth it to test this one module more thoroughly. Please either switch to using compiled binary or please remove testing with real binaries completely.